src/main/java/com/metabug/web/controller/TicketController.java

```diff
@@ -29,16 +39,26 @@ public ModelAndView getView() {
 
         modelAndView.setViewName("add-ticket");
         modelAndView.addObject("ticketDto", ticketDto);
+        modelAndView.addObject("assignees", userService.findAll());
```

Comment:
why assignees and not developers to keep it consistent with what we name it in DB?

Reply:
Removed, because we're not setting assignee in `/add-ticket`